src/app/globals.css

```
@@ -120,3 +120,8 @@
     @apply bg-background text-foreground;
   }
 }
+
+::view-transition-old(root), ::view-transition-new(root) {
```

Biome (2.1.2)

[error] Unknown type selector is not allowed.

See MDN web docs for more details.
Consider replacing the unknown type selector with valid one.

(lint/correctness/noUnknownTypeSelector)